go.mod

```
@@ -10,9 +10,12 @@ require (
 )
 
 require (
-	// Requires latest version for automated release process
-	github.com/cyberark/conjur-opentelemetry-tracer v0.0.1-336
+	// Version number used here is ignored
+	github.com/cyberark/conjur-opentelemetry-tracer v1.55.55
 	github.com/davecgh/go-spew v1.1.1 // indirect
 )
 
 replace gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c => gopkg.in/yaml.v3 v3.0.1
+
+// DO NOT EDIT: CHANGES TO THE BELOW LINE WILL BREAK AUTOMATED RELEASES
```

if `go mod tidy` will update this, I would suggest making this comment worded even more clearly.
```suggestion
// DO NOT EDIT: CHANGES TO THE BELOW LINE WILL BREAK AUTOMATED RELEASES. IF THE LAST
// WORD IS ANYTHING OTHER THAN "latest", CHANGE IT BACK.
```